plugins/org.eclipse.sirius.tests.junit/src/org/eclipse/sirius/tests/unit/diagram/migration/WorkspaceImageGMFBoundsMigrationParticipantTest.java

```diff
@@ -89,11 +89,13 @@ public void testGMFBounds() {
         validateNodeSize("enum", DNode.class, 50, 50, false); //$NON-NLS-1$
         validateNodeSize("att", DNode.class, 1, 1, false); //$NON-NLS-1$
         validateCollapseSize("att", DNode.class, 10, 10); //$NON-NLS-1$
-        validateNodeSize("P1", DNode.class, 100, 70, false); //$NON-NLS-1$
+        validateNodeSize("p1", DNode.class, 100, 70, false); //$NON-NLS-1$
         validateNodeSize("container_image", DNodeContainer.class, 160, 48, true); //$NON-NLS-1$
         validateNodeSize("list_image", DNodeList.class, 160, 48, true); //$NON-NLS-1$
         validateNodeSize("cont_not_found", DNodeContainer.class, 100, 70, true); // Container default size //$NON-NLS-1$
         validateNodeSize("list_not_found", DNodeList.class, 100, 70, true); // Container default size //$NON-NLS-1$
+        validateNodeSize("p2", DNode.class, 40, 12, false); // width = 40 as the number of EClass and //$NON-NLS-1$
```

Comment:
Note that `$NON-NLS-1$` is no longer required in tests

Reply:
Yes but it was already like this in this test.